### ISSUE TITLE
Sidenav auto-close on Nexus 5, Android5.1

### ISSUE THREAD

**Issue by androidfreakers** [angular/material#2477]
I have created a sample application to understand sidenav functionality. To test that on nexus 5 it has a wired issue.
When i click on button to toggle the side nav. It opens the side nav and closes immediately.

Here is sample code.
HTML --->
`<body ng-app="PerformanceEvaluation">
        <div layout="row" layout-fill>
            <md-sidenav md-component-id="left" class="md-sidenav-left" >
                <md-content ng-controller="categoryController">
                    <md-list>
                        <md-subheader>Categories</md-subheader>
                        <md-list-item>
                            <div layout="row" layout-align="start center" style="padding:0 10px;" ng-repeat="category in categories">
                                <img ng-src="{{category.icon}}"/>
                                <p style="margin-left:10px;">{{category.title}}</p>
                            </div>
                        </md-list-item>
                    </md-list>
                </md-content>
            </md-sidenav>
            <div ng-controller="test">
                <md-button ng-click="toggleSideNav()">Toggle Side Nav</md-button>
            </div>
        </div>
    </body>`

js ---> 

```'use strict';
angular.module("PerformanceEvaluation", ['ngMaterial','PerformanceEvaluation.categories'])
.config(["$mdThemingProvider","$mdIconProvider",function ($mdThemingProvider, $mdIconProvider) {
    $mdThemingProvider.theme('default').primaryPalette('red').accentPalette('cyan').warnPalette('green');
}])
.controller("test",['$scope','$mdSidenav', function($scope, $mdSidenav){
    $scope.toggleSideNav = function(){
        if($mdSidenav("left").isOpen()){
            $mdSidenav("left").close();
        }
        else{
            $mdSidenav("left").open();
        }
    }
}]);```

to fill  the cotegories. --->

`'use strict';
angular.module("PerformanceEvaluation.categories",[])
.controller("categoryController",['$scope',function($scope){
    $scope.categories = [
        {
            "icon":'bower_components/material-design-icons/action/1x_web/ic_3d_rotation_black_18dp.png',
            "title":"tests"
        },
        {
            "icon":'bower_components/material-design-icons/action/1x_web/ic_3d_rotation_black_18dp.png',
            "title":"tests"
        },
        {
            "icon":'bower_components/material-design-icons/action/1x_web/ic_3d_rotation_black_18dp.png',
            "title":"tests"
        }
    ]
}])`

**Comment by xeor** [comment]
This happens on my Nexus 6 as well..

**Comment by marekmicek** [comment]
Which version of angular material are you using ? I think it should be fixed in 0.9 (0.8.3 have this issue though).

angular/angular.js#6251

**Comment by xeor** [comment]
This is fixed in 0.9, just tested :)
